core/src/test/java/com/microsoft/applicationinsights/internal/statsbeat/StatsbeatTestUtils.java

```diff
@@ -78,6 +78,8 @@
         FEATURE_MAP_DECODING.put(3, Feature.JAVA_VENDOR_ADOPT_OPENJDK);
         FEATURE_MAP_DECODING.put(4, Feature.JAVA_VENDOR_REDHAT);
         FEATURE_MAP_DECODING.put(5, Feature.JAVA_VENDOR_OTHER);
+        FEATURE_MAP_DECODING.put(6, Feature.AAD_ON);
+        FEATURE_MAP_DECODING.put(7, Feature.AAD_OFF);
```

Comment:
do we need two features for this, or can we just track "ON"?

Reply:
we need both.. similar to java vendor. we need to track all the possible vendors. Key is static.